README.md

```diff
@@ -232,11 +232,7 @@ import createLogger from 'redux-logger';
 
 const actionTransformer = action => {
   if (action.type === 'BATCHING_REDUCER.BATCH') {
-    action.payload.type = action.payload.reduce((result, next) => {
-      const prefix = result ? result + ' => ' : '';
-      return prefix + next.type;
-    }, '');
-
+    action.payload.type = action.payload.map(result => result.type).join(' => ');
```

Comment:
Looks fine. Thanks.

But you should rename the parameter of predicate from `result` to `next`:

```javascript
action.payload.map(next => next.type)
```

Do it, please, and then we'll merge it.